spec/spec.md

```diff
@@ -5,22 +5,25 @@
 **Specification Status:** Draft
 
 **Editors:**
+~ [Orie Steele](https://www.linkedin.com/in/or13b/) (Transmute)
```

Comment:
I am happy to be listed as an editor, but I think at least 1 person from Bloom should be on the hook for this working out....

@JaceHensley can you circle waggons with Affinidi and Bloom and get someone committed to fielding a continious contributor / editor to this work?

it might be you...

Reply:
I'm happy to adjust names however folks feel is appropriate.